modules/search/pages/import-search-index.adoc

```diff
@@ -14,25 +14,57 @@ For more information about how to change Services on your database, see xref:clo
 
 * You have a bucket with scopes and collections in your database. 
 For more information, see xref:cloud:clusters:data-service/manage-buckets.adoc[].
+
+* You have a Search index or Search index alias definition saved as a JSON file.
+For more information about the properties you can include in a Search index or index alias definition, see xref:search-index-params.adoc[].
  
 * You have logged in to the Couchbase {page-ui-name}. 
 
-== Procedure
 
-To import a xref:create-search-indexes.adoc[Search index definition] or xref:index-aliases.adoc[Search alias] with the {page-ui-name}:
+== Import a Search Index Definition 
+
+To import a full xref:search-index-params.adoc[Search index definition] with the {page-ui-name}: 
+
+. On the *Databases* page, select the database where you want to import a JSON Search index definition.
```

Comment:
Alternative suggestion:
On the Databases page, select the target database for the import.

Reply:
I think I'd rather keep it explicit like this.